### PR TITLE
Added check for `options` in `authzHttpClient`.

### PR DESCRIPTION
**Pull request by mandyvenables**

### CODE REVIEW

**Review by dmitrizagidulin — APPROVED** [review]
👍

**Comment by dmitrizagidulin** [comment]
(This is for cases like `httpClient.get(url)`, where there's no options object)

File: lib/authzHttpClient.js [hunk 1]
```diff
@@ -45,7 +45,12 @@ async function _handleRequest(target, thisArg, args, oAuth2Client) {
 
   // create the authz header using the token
   // add the authz header to the headers object of args
-  const [url, options] = args;
+  let options = {};
+
+  const [url] = args;
+  if(args[1]) {
+    options = args[1];
+  }
```
Comment (dlongley) [inline lib/authzHttpClient.js:53]: Can we just use defaults? Like this:

```js
const [url, options = {}] = args;
```

**Review by dlongley — CHANGES_REQUESTED** [review]

File: lib/authzHttpClient.js (hunk 1 above)
Comment (mandyvenables) [reply lib/authzHttpClient.js:53]: That looks much better, updated here: https://github.com/digitalbazaar/bedrock-oauth2-client/pull/10

**Pull request merged**